handsontable/src/plugins/collapsibleColumns/collapsibleColumns.js

```diff
@@ -69,6 +70,35 @@ const actionDictionary = new Map([
  *   ],
  * });
  * ```
+ * :::
+ *
+ * ::: only-for react
+ * ```jsx
+ * <HotTable
+ *   data={generateDataObj()}
+ *   colHeaders={true}
+ *   rowHeaders={true}
+ *   // @ts-ignore
+ *   nestedHeaders={true}
+ *   // enable plugin
+ *   collapsibleColumns={true}
+ * />
+ *
+ * // or
+ * <HotTable
+ *   data={generateDataObj()}
+ *   colHeaders={true}
+ *   rowHeaders={true}
+ *   // @ts-ignore
```

Comment:
Same as at `:81`

Reply:
We can discuss it here: https://github.com/handsontable/handsontable/pull/9791#discussion_r951117044